project/roadmap.md

```diff
@@ -39,10 +39,10 @@ A tick indicates an item is implemented and has automated tests in place.
 
 #### Database API
 
-- [ ]	GET/_api/database List of databases
-- [ ]	POST/_api/database Create database
+- [X]	GET/_api/database List of databases
+- [X]	POST/_api/database Create database
```

Comment:
Do we have tests yet for Create database?

Reply:
I'll answer my own question - as far as I can tell, we haven't done the proper implementation for Create database yet so we shouldn't check it off in the task list yet. Can you uncheck it for now?

Reply:
Yep I will do that.

Reply:
While we do not have an explicit test for create database, most (if not all) the per-test-class fixtures use the method the create a test database.

Reply:
Yes, the criteria for marking a task complete is that we have implemented both the endpoint and tests.